### PR TITLE
[Plugin] More plugin APIs 🔌

### LINKED ISSUE #13078: [Plugin] Add colour picker widget

### LINKED ISSUE #13431: [Plugin] UI disabled widgets can still be interacted with
**OS:** Windows 10
**Version:** 0.3.2
**Commit/Build:** cea5fab

When you disable widgets using the 'isDisabled' property, these widgets can still be interacted with and will still trigger events. While for the spinner the issue could be handled by plugin code, but for the dropdown widget there's no way of stopping the dropdown from opening. (See attached screenshot.)

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
1. Disable a widget via 'widget.isDisabled = true'.
2. Interact with it.

**Screenshots / Video:**
afbeelding

### LINKED ISSUE #13495: [Plugin] Add properties for park value, guests and company value
Current plugin API does not support to get park's various stats, such as the total number of guests, park value or company value while park rating is easy to access via ``park.rating``. It will be great if we can access them like that.

- [ ] Total number of guests
- [ ] Park value
- [ ] Company value (could be omitted since it is a simple substraction, imo)

### LINKED ISSUE #13510: [Plugin] list view scroll resets when items is set

### LINKED ISSUE #13512: [Plugin] Add item separators to list view

### PR DESCRIPTION
**Pull request by IntelOrca**
* Fix #13495: Add properties for park value, guests and company value
* Add more park attributes to plugin API
* Fix #13431: [Plugin] UI disabled widgets can still be interacted with.
* Fix #13078: [Plugin] Add colour picker widget
* Fix: #13509 [Plugin] Add ability to format strings
* Fix #13510: [Plugin] list view scroll resets when items is set.
* Fix #13512: [Plugin] Add item separators to list view

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 - Feature: [#13376] Open custom window at specified tab.
 - Feature: [#13398] Add pause button to the Track Designer.
+- Feature: [#13495] [Plugin] Add properties for park value, guests and company value.
 - Change: [#13346] Change FootpathScenery to FootpathAddition in all occurrences.
 - Fix: [#12895] Mechanics are called to repair rides that have already been fixed.
 - Fix: [#13257] Rides that are exactly the minimum objective length are not counted.
 - Fix: [#13334] Uninitialised variables in CustomTabDesc.
 - Fix: [#13342] Rename tabChange to onTabChange in WindowDesc interface.
 - Fix: [#13427] Newly created Go-Karts show "Race won by <blank>".
+- Fix: [#13431] [Plugin] UI disabled widgets can still be interacted with.
```
Comment (duncanspumpkin) [inline distribution/changelog.txt:14]: should you perhaps add the plug in tag on the other existing fixes?

Comment (IntelOrca) [reply distribution/changelog.txt:14]: 🤔

File: src/openrct2-ui/scripting/CustomListView.h [hunk 2]
```diff
@@ -52,6 +52,7 @@ namespace OpenRCT2::Ui::Windows
 
     struct ListViewItem
     {
+        bool IsSeperator{};
```
Comment (tupaschoal) [inline src/openrct2-ui/scripting/CustomListView.h:55]:
```suggestion
        bool IsSeparator{};
```

Guess it should read "separator" all across the PR?

File: distribution/changelog.txt (hunk 1 above)
Comment (IntelOrca) [reply distribution/changelog.txt:16]: All changes to changelog in this PR have `[Plugin]`, so changing others can be done after this is merged.

**Pull request merged**